src/components/document/editable-document-content.tsx

```diff
@@ -1,40 +1,40 @@
-import React, { useContext, useRef } from "react";
+import React, { useContext, useRef, useState } from "react";
 import classNames from "classnames";
-import { AppConfigContext, IAppConfigContext } from "../../app-config-context";
+import { AppConfigContext } from "../../app-config-context";
 import { CanvasComponent } from "./canvas";
 import { DocumentContextReact } from "./document-context";
 import { FourUpComponent } from "../four-up";
 import { useDocumentContext } from "../../hooks/use-document-context";
 import { useGroupsStore, useUIStore, useUserStore } from "../../hooks/use-stores";
-import { ToolbarComponent, ToolbarConfig } from "../toolbar";
+import { ToolbarComponent } from "../toolbar";
 import { EditableToolApiInterfaceRef, EditableToolApiInterfaceRefContext } from "../tools/tool-api";
 import { DocumentModelType } from "../../models/document/document";
 import { ProblemDocument } from "../../models/document/document-types";
 import { WorkspaceMode } from "../../models/stores/workspace";
-import { getToolContentInfoByTool } from "../../models/tools/tool-content-info";
-import { IToolButtonConfig } from "../tool-button";
+import { clone } from "mobx-state-tree";
```

Comment:
Conventionally, we've been putting third-party imports at the top, so this would go up with "react" and "classnames".

Reply:
I don't suppose you have a VSCode plugin that applies these rules when I use the automatic adding of imports?

Reply:
I see we could enforce it with eslint:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md